persistence/block.go

```diff
@@ -44,30 +45,56 @@ func (p PostgresContext) TransactionExists(transactionHash string) (bool, error)
 	return false, nil
 }
 
-func (p PostgresContext) StoreTransaction(transactionProtoBytes []byte) error {
-	log.Println("TODO: StoreTransaction not implemented")
+func (p *PostgresContext) StoreTransaction(transactionProtoBytes []byte) error {
+	p.currentBlockTxs = append(p.currentBlockTxs, transactionProtoBytes)
```

Comment:
This is confusing - at this part of the lifecycle, I'm not convince that the `currentBlock` wouldn't already be populated with `transactions`

Reply:
I'm also not excited that this is being worked on before https://github.com/pokt-network/pocket/pull/302

Notice this lifecycle and consider conforming to it

Reply:
I apoligize. Wasn't my intention.

Per my comment in https://github.com/pokt-network/pocket/pull/302#pullrequestreview-1146448248, I'm pausing work on this until we discuss the end-to-end flow and get #302 in.